src/cryptonote_core/cryptonote_tx_utils.cpp

```diff
@@ -232,12 +231,35 @@ namespace cryptonote
       }
       summary_inputs_money += src_entr.amount;
 
+      // disposable addressing scheme
+      boost::optional<keypair> disposable_key;
+      {
+        char data[HASH_SIZE + 8];
+        memcpy(data, &sender_account_keys.m_view_secret_key, HASH_SIZE);
```

Comment:
`HASH_SIZE` and `sizeof(m_view_key_secret_key)` are identical, but its probably better if `sizeof` is used for the calculations.